types/lodash/lodash-tests.ts

```diff
@@ -3660,21 +3660,27 @@ fp.now(); // $ExpectType number
         leading: true,
         trailing: false,
     };
+    const optionsNoLeading: _.ThrottleSettingsNoLeading = {
```

Comment:
It might be worth writing the options inline, instead of explicitly setting the type here, since most users won't set an explicit type, and that could mess with the overload selection.